src/web/mjs/connectors/Qimiaomh.mjs

```diff
@@ -0,0 +1,84 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class EpikManga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'qimiaomh';
+        super.label = '奇妙漫画 (Qimiaomh)';
+        this.tags = [ '!test', 'manhua', 'chinese' ];
+        this.url = 'https://www.qimiaomh.com';
+
+        this.queryMangas = 'div.wrapper div.mt20 div.classification h2 a';
+        this.queryChapters = 'div.comic-content-list ul.comic-content-c li.tit a';
+        this.queryPages = 'div.cartoon div.loadimg img.man_img';
+    }
+
+    async _getMangasFromPage(page) {
+        const uri = new URL('/list-1------updatetime--' + page + '.html', this.url);
+        const request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryMangas);
+        let checkData = await this.fetchDOM(request, 'div.paging ul li a');
+        let result = {
+            data: null,
+            run: true
+        };
+
+        result.data = data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.text.trim()
+            };
+        });
+
+        result.run = checkData.find(element => {
+            return element.href == "javascript:void(0)";
+        });
+
+        if(page === 1)result.run = null;
```

Comment:
May use short-circuiting to combine conditions instead of a separate conditional check
`result.run = page === 1 || checkData.find(...);`

Reply:
need to override check explicitly for the first page